monarchmoney/monarchmoney.py

```diff
@@ -404,6 +404,59 @@ async def get_transactions(
             operation="GetTransactionsList", graphql_query=query, variables=variables
         )
 
+    async def create_transaction(
+        self,
+        date: str,
+        account_id: str,
+        amount: float,
+        merchant_name: str,
+        category_id: str,
+        notes: str = ""
+    ) -> Dict[str, Any]:
+        """
+        Creates a transaction with the given parameters
+        """
+        query = gql(
+            """
+      mutation Common_CreateTransactionMutation($input: CreateTransactionMutationInput!) {
```

Comment:
nit: fix indentation

Reply:
Happy to do so. It looks like there may be some inconsistent indentation:

```
get_subscription_details (+ all others other than get_transactions)
-------

        query = gql(
            """
      query GetSubscriptionDetails {
        subscription {
          id
          paymentSource
          referralCode
          isOnFreeTrial
          hasPremiumEntitlement
          __typename
        }
      }
    """
        )
```

```
get_transactions
-------

        query = gql(
            """
          query GetTransactionsList($offset: Int, $limit: Int, $filters: TransactionFilterInput, $orderBy: TransactionOrdering) {
            allTransactions(filters: $filters) {
              totalCount
              results(offset: $offset, limit: $limit, orderBy: $orderBy) {
                id
                ...TransactionOverviewFields
                __typename
              }
              __typename
            }
            transactionRules {
              id
              __typename
            }
          }
    
          fragment TransactionOverviewFields on Transaction {
            id
            amount
            pending
            date
            hideFromReports
            plaidName
            notes
            isRecurring
            reviewStatus
            needsReview
            attachments {
              id
              __typename
            }
            isSplitTransaction
            category {
              id
              name
              icon
              __typename
            }
            merchant {
              name
              id
              transactionsCount
              __typename
            }
            account {
              id
              displayName
              __typename
            }
            tags {
              id
              name
              color
              order
              __typename
            }
            __typename
          }
        """
        )
```

Which one is the desired indentation? I can make them consistent across the class. My guess is the indentation used in `get_transactions` is the desired indentation.

Apologies if you're referring to some other thing I'm missing, just let me know.

Reply:
Thanks for the call out!  I like the way you're going in the latest commit and we'll stick with that :)